### ISSUE TITLE
David Jones feedback

### ISSUE THREAD

**Issue by Godley** [Godley/Music-Library#276]
## 1 Introduction

3rd para "it's" -> "its" (twice)
## 3.2.1

4th para (starting "Finally, due to the language"). Github is not an
"Open Source Code Repository", I would just describe it as an "online
source code hosting service". (Aside: back in the day, googlecode
_required_ hosted projects to be open source and you had to pick a
license; it was part of github's genius to drop the requirement that a
project be open source).

page 11, paragraph starting "Furthermore, as it is dynamically typed":
"it's" -> "its". I hate english.

paragraph starting "As exemplified", at the end of this, replace
"Python when the project is open sourced" with "Python, so a large
available community of potential developers for when the project is
open sourced".
## 4.1 User Interface Design

"The User interface for this project is designed after looking at the
user interfaces used by other music appli-cations."

"survey" is better than "looking at", so I suggest: "I surveyed other
music applications, with particular regard to their user interfaces".

"hand drawn designs" -> "wireframes"

Swop the order of figures 16 and 18. That puts the finished interface
first (looks good), then shows the breakdown in figure 17, then shows
the primary wireframe design for this interface. You'll have to fiddle
the text.
## 4.2 Test Design and System Testing

instead of "no test design or plan was produced" I suggest "the test
design evolves incrementally in parallel with the implementation of
features; the test plan is interwoven with the implementation". The
sentence is a bit too long anyway and needs breaking up with a full
stop.
### 4.2.2

"python" should be "Python" (probably throughout, but this is the
first place I spotted it).

Last paragraph "this enabled the developer to check" -> "continuous
testing enabled the developer to check".
## 4.2.3

second para: "ToLily" should be in \tt (fixed width font)
## 4.2.4

typo: "hierarcy" should be "hierarchy".
## 4.3.3

After "Model View Controller" put "(MVC)" so you can legitimately
refer to it later.
## 4.3.7

I'm pretty sure "sub class" shold be spelt "subclass"
## 4.4.2

last paragraph, sentence starting "It may have been a suggestion to
extend" This is a very negative phrasing. I think a more positive spin
is to say something like "No finite research period would be long
enough to fully discover the ramifications of all possible formats.
Therefore a fixed period of research was undertaken, with the
understanding that further features of the particular formats used
would be discovered during development."
## 4.4.3

extra "s" on the end of "lock to one platform.s".

last paragraph: "it's" -> "its" (in the paragraph starting "The third
and final option"
## 5.2.2

You are selling yourself short. My reading of the secondary objectives:
- audio playback: not attempted, but clearly not high risk. No
  challenges expected to any future implementor.
- MusicOCR conversion: while OCR has not been attempted, any OCR
  component would be able to use the metadata and object model that has
  already been developed for this project.
- Difficulty rating: not implemented, but some "scoping out" work
  done, and a survey designed and executed which would be necessary for
  future work that builds a predictive model to assess difficulty
  rating.

so I would add a sentence saying "No secondary objective has been
finished, but the current project has made useful progress towards
some of them" and then summarise the above.
## 5.2.3

2nd para: you'll definitely need a ref for the claim "the highest user
count of any operating system", or just make it more vague: "the
target audience is more likely to have a Windows computer than any
other". (I have a sneaking suspicion that Android (hence, Linux) has
the highest user count by the way).
## 5.3

I like this section.
## 6

5th para: "outpus" -> "outputs"

I think what you have achieved here is a bridge between musicians and
software developers. I view it either as a uniting piece joing these
two communities together, or a nucleus around which the nascent
community of "musicians that can program" can group.
